### PR TITLE
Add Layton's Mystery Journey: Katrielle and the Millionaires' Conspiracy

### PR DESCRIPTION
**Pull request by Macley-Kun**
Adds Layton's Mystery Journey: Katrielle and the Millionaires' Conspiracy.

*Automatically generated by PR builder*

### CODE REVIEW

**Comment by Macley-Kun** [comment]
Note that there is a jpn version out there!
I sadly didn't find the copy of the game, but wanted to mention it anyhow here 😛

File: games/laytons-mystery-journey-katrielle-and-the-millionaires-conspiracy/game.dat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+title = "Layton's Mystery Journey: Katrielle and the Millionaires' Conspiracy"
+description = "Layton's Mystery Journey: Katrielle and the Millionaires' Conspiracy is a puzzle video game developed by Level-5. It is the seventh main entry in the Professor Layton series and follows a new protagonist, Katrielle Layton.\nKatrielle \"Kat\" Layton, who solves puzzles in her father's place alongside her talking dog Sherl and her friends Emiliana Perfetti and Ernest Greeves. When her father vanishes, Katrielle goes off in search of him, coming across various puzzles and mysteries along the way."
```
Comment (Hexagon12) [inline games/laytons-mystery-journey-katrielle-and-the-millionaires-conspiracy/game.dat]: Delete the unnecessary `\n` and `\`

**Comment by Macley-Kun** [comment]
Editted https://github.com/citra-emu/citra-games-wiki/wiki/laytons-mystery-journey-katrielle-and-the-millionaires-conspiracy/
with a valid summary. Sorry i should've checked this more troughly.

File: games/laytons-mystery-journey-katrielle-and-the-millionaires-conspiracy/game.dat [hunk 2]
```diff
@@ -0,0 +1,19 @@
+title = "Layton's Mystery Journey: Katrielle and the Millionaires' Conspiracy"
+description = "Layton's Mystery Journey: Katrielle and the Millionaires' Conspiracy is a puzzle video game developed by Level-5. It is the seventh main entry in the Professor Layton series and follows a new protagonist, Katrielle Layton. Katrielle "Kat" Layton, who solves puzzles in her father's place alongside her talking dog Sherl and her friends Emiliana Perfetti and Ernest Greeves. When her father vanishes, Katrielle goes off in search of him, coming across various puzzles and mysteries along the way."
```
Comment (Leo626) [inline games/laytons-mystery-journey-katrielle-and-the-millionaires-conspiracy/game.dat]: `"Kat"` needs to be escaped like this `\"Kat\"`.

Comment (Macley-Kun) [reply games/laytons-mystery-journey-katrielle-and-the-millionaires-conspiracy/game.dat]: Then i'll add them back haha

[the remark this answers is not in the document]
Comment (Macley-Kun) [reply games/laytons-mystery-journey-katrielle-and-the-millionaires-conspiracy/game.dat]: Fixed this by following Leo626's suggestion.
> "Kat" needs to be escaped like this \"Kat\".

**Comment by Macley-Kun** [comment]
Anything left captains? :')

**Pull request merged**